src/components/ProductTabs/ProductTabs.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { FC } from "react";
+import { ICard } from "@models/ICard";
+import { Tabs, TabLink, TabContent } from "react-tabs-redux";
+import s from "./ProductTabs.module.scss";
+
+const ProductTabs: FC<{ product: ICard }> = ({ product }) => {
+  const styles = {
+    activeLinkStyle: {
+      borderBottom: "2px solid #333",
+    },
+    visibleTabStyle: {
+      display: "inline-block",
+    },
+  };
```

Comment:
Don't like styles inside component

Reply:
and color instead of variable

Reply:
I know, but I don't see a better option, <Tabs> accepts only these parameters with the object type, since the developer did not take care of this point

Reply:
Yes, I understand. I transferred everything I could to css , but these two variables activeLinkStyle and visibleTabStyle in the Tabs element can only accept an object, so I couldn’t move them to .scss .

Reply:
this decision was made by the developer of the library (

Reply:
I can somehow add it to the styles with these conditions ?

Reply:
Changed